### PR TITLE
Fixes recyclers removing brainmobs from holders, removes direct loc setting and src.loc in playsound too

### LINKED ISSUE #31146: Putting a brain into a recycler ejects the brainmob
Roundid: 76793
Recycling a brain left the brain, yet also ejected the invisible, realnamed brainmob.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #31146

### CODE REVIEW

File: code/game/machinery/recycler.dm [hunk 1]
```diff
@@ -137,25 +138,25 @@
...
 /obj/machinery/recycler/proc/reboot()
-	playsound(src.loc, 'sound/machines/ping.ogg', 50, 0)
+	playsound(src, 'sound/machines/ping.ogg', 50, 0)
 	safety_mode = FALSE
 	update_icon()
 
 /obj/machinery/recycler/proc/crush_living(mob/living/L)
 
-	L.loc = src.loc
+	L.loc = src
```
Comment (Cyberboss) [inline code/game/machinery/recycler.dm]: forceMove

File: code/game/machinery/recycler.dm [hunk 2]
```diff
@@ -137,25 +138,25 @@
 
 
 /obj/machinery/recycler/proc/emergency_stop(mob/living/L)
-	playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 50, 0)
+	playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 0)
 	safety_mode = TRUE
 	update_icon()
-	L.loc = src.loc
+	L.forceMove(src.loc)
```
Comment (Cyberboss) [inline code/game/machinery/recycler.dm]: remove src

File: code/game/machinery/recycler.dm [hunk 3]
```diff
@@ -107,24 +107,25 @@
...
+			playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 0)
+			AM.forceMove(src.loc)
 
 	if(items_recycled && sound)
-		playsound(src.loc, item_recycle_sound, 50, 1)
+		playsound(src, item_recycle_sound, 50, 1)
 
 /obj/machinery/recycler/proc/recycle_item(obj/item/I)
-	I.loc = src.loc
+	I.forceMove(src.loc)
```
Comment (Cyberboss) [inline code/game/machinery/recycler.dm]: remove src

File: code/game/machinery/recycler.dm [hunk 4]
```diff
@@ -107,24 +107,25 @@
...
 			else
 				emergency_stop(AM)
 		else if(istype(AM, /obj/item))
 			recycle_item(AM)
 			items_recycled++
 		else
-			playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 50, 0)
-			AM.loc = src.loc
+			playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 0)
+			AM.forceMove(src.loc)
```
Comment (Cyberboss) [inline code/game/machinery/recycler.dm]: remove src

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by kevinz000** [comment]
How the fuck did I miss the src.s when I SPECIFICALLY REMOVED SRCS REEE!!!

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**